fetch.bs

```diff
@@ -1165,6 +1165,12 @@ Unless stated otherwise, it is unset.
 
 <p class="note no-backref">This flag is for exclusive use by HTML's navigate algorithm. [[!HTML]]
 
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=concept-request-history-navigation-flag>history-navigation flag</dfn>.
+Unless stated otherwise, it is unset.
+
+<p class="note no-backref">This flag is for exclusive use by HTML's navigate algorithm. [[!HTML]]
```

Comment:
We should move this and the flag before it, up a bit. See #732. Sorry for not noticing that earlier. I can probably take care of this as a fixup commit once everything is close to landing, but I'm also happy for you to take care of it.

Reply:
Done.